django/contrib/admin/static/admin/css/base.css

```diff
@@ -538,6 +539,7 @@ a.button {
     border: none;
     font-weight: 400;
     background: var(--default-button-bg);
+    color: var(--default-button-fg);
```

Comment:
Do we need to change it? It's not in the ticket description and it already has AA contrast: 
![image](https://github.com/django/django/assets/2865885/e8394ebf-7336-43aa-b7fa-690a6a5b7d78)